### ISSUE TITLE
NetBSD support

### ISSUE THREAD

**Issue by aerth** [creack/pty#38]
```
  I tried GOOS=netbsd GOARCH=amd64 ./mktypes.bash

  Added an "echo Complete" to the end to make sure the script ran fully, it
  ran. I didn't get a new _ztypes_netbsd_amd64.go

  How do I generate the netbsd ztype file?

  thanks!
```

**Comment by paulzhol** [comment]
only FreeBSD needs  a ztypes_$GOOS_$GOARCH.go, for a definition of an ioctl argument of type fiodgname_arg:
```
struct fiodgname_arg {
	int	len;
	void	*buf;
};
```
Look at types_freebsd.go and pty_freebsd.go:45, NetBSD might have a very similar implementation.

**Comment by yob** [comment]
Maybe this can be closed now the #117 has merged?
